src/pages/home/report/ReportActionItemMessage.js

```diff
@@ -40,6 +41,20 @@ function ReportActionItemMessage(props) {
     const styles = useThemeStyles();
     const fragments = _.compact(props.action.previousMessage || props.action.message);
     const isIOUReport = ReportActionsUtils.isMoneyRequestAction(props.action);
+    if (ReportActionsUtils.isChannelLogMemberAction(props.action)) {
```

Comment:
In my opinion, we should rename it consistently with our pattern. It would be: `isMemberChangeAction`. I think I included this in the list I shared before.

Thanks for your patience, this review has had many rounds. I hope you see some value in the suggested changes.